core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -100,6 +100,18 @@ class LocalEntityProvider(workspace: Workspace, implicit protected val dataSourc
     }
   }
 
+  override def deleteEntityAttribute(workspaceContext: Workspace, entityType: String, entityAttributeNamespace: String, entityAttributeName: String): Future[Vector[Int]] = {
+    dataSource.inTransaction { dataAccess =>
+      dataAccess.entityQuery.checkAttributeNameExists(workspaceContext.workspaceIdAsUUID, entityType, entityAttributeNamespace, entityAttributeName) flatMap { attributeNameExists =>
+        if (attributeNameExists.toList.head)
+          dataAccess.entityQuery.deleteColumn(workspaceContext, entityType, entityAttributeNamespace, entityAttributeName)
+        else
+          throw new DataEntityException(message = s"Could not find column $entityAttributeName belonging to entity namespace $entityAttributeNamespace of entity type $entityType in workspace ${workspaceContext.namespace}/${workspaceContext.name}",
+            code = StatusCodes.BadRequest)
+      }
+    }
+  }
```

Comment:
do you need the `checkAttributeNameExists` at all? I believe `deleteColumn` returns the number of rows it actually deleted. Could you run the delete and inspect its results? If it deleted > 0 rows, all good; if it deleted 0 rows, then throw the `Could not find column …` exception.

Reply:
This is a good point! I will do this instead